File: app/src/main/java/ch/heigvd/pro/b04/android/Poll/PollViewModel.java
package ch.heigvd.pro.b04.android.Poll;

import android.app.Application;

import androidx.annotation.NonNull;

public class PollViewModel extends SharedViewModel {
    public PollViewModel(@NonNull Application application, String token, int idModerator, int idPoll) {
        super(application, token, idModerator, idPoll);
    }
}

File: app/src/main/java/ch/heigvd/pro/b04/android/Poll/PollViewModel.kt
package ch.heigvd.pro.b04.android.Poll

import android.app.Application
import androidx.lifecycle.MutableLiveData
import ch.heigvd.pro.b04.android.Datamodel.Question
import ch.heigvd.pro.b04.android.Network.RequestsViewModel

class PollViewModel(application: Application,
                    idModerator : Int,
                    idPoll : Int,
                    token : String
) : RequestsViewModel(application, idModerator, idPoll, token) {
    val questionToView : MutableLiveData<Question> = MutableLiveData()

    fun goToQuestion(question: Question) {
        questionToView.postValue(question)
    }

}
